projects/kit/directives/highlight/highlight.options.ts

```diff
@@ -0,0 +1,13 @@
+import {tuiCreateToken} from '@taiga-ui/cdk/utils/miscellaneous';
```

Comment:
https://github.com/taiga-family/taiga-ui/blob/main/projects/kit/directives/password/password.options.ts

please use that approach

Reply:
done [d115631](https://github.com/taiga-family/taiga-ui/pull/8812/commits/d115631434d97f64d18c0dbb32590ed6ed2b1660)

<img width="453" alt="image" src="https://github.com/user-attachments/assets/b1d055cd-bfbb-4a4c-98d9-1d5e5c054528">